CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 7.70
 -----
+*   Updates
```

Comment:
I have a question. Should the seek action pause the audio? Currently, if we are playing an episode and try o seek the audio is not paused, but if I try to see in clip sharing the audio is paused. See the video:

https://github.com/user-attachments/assets/081ded78-1462-47fd-87d6-cbef74f98b00

Reply:
Yes, I deliberately pause the audio.